ee_tests/cico_run_e2e_tests.sh

```diff
@@ -106,6 +109,8 @@ docker exec $CONTAINER_NAME bash -c 'cp ./target/screenshots/* ./e2e/${JOB_NAME}
 docker exec $CONTAINER_NAME bash -c 'cp ./target/zabbix/* ./e2e/${JOB_NAME}/${BUILD_NUMBER}'
 docker exec $CONTAINER_NAME rsync --password-file=./password_file -PHva --relative ./e2e/${JOB_NAME}/${BUILD_NUMBER}  devtools@artifacts.ci.centos.org::devtools/
 
+docker exec fabric8-test zabbix_sender -vv -T -i zabbix-report.txt -z $ZABBIX_SERVER -p $ZABBIX_PORT;
```

Comment:
1. change `fabric8-test` to `$CONTAINER_NAME`
2. is `zabbix-report.txt` in top-level directory or in `target/zabbix`?
3. do you need semicolon at the end of line?